src/get-started/create-service.md

```diff
@@ -492,13 +520,13 @@ With this aim we declare a blank struct that includes a set of methods with the
 following signature:
 
 ```rust
-fn my_method(state: &ServiceApiState, query: MyQuery) -> api::Result<MyResponse>
+fn my_method(state: &ServiceApiState<'_>, query: MyQuery) -> api::Result<MyResponse>
 ```
 
 The `state` contains a channel, i.e. a connection to the blockchain node
```

Comment:
I think describing the channel is no longer relevant because it's not used by the service.